Use environment markers for conditional dependencies

jdufresne opened the pull request:
Using environment markers has support among modern Python packaging tools (setuptools, pip, wheel, etc.). Allows describing environment specific dependencies in setup.py without using runtime logic.

giampaolo commented:
Good stuff. Can't merge it from my phone. I approved the PR so I think you should be able to.

jdufresne commented:
Thanks. I'd be glad to merge, but I don't have merge rights on the repo. :smile:

giampaolo commented:
I will merge it later then ;)

On Sun, 4 Mar 2018 at 20:02, Jon Dufresne <notifications@github.com> wrote:

> Thanks. I'd be glad to merge, but I don't have merge rights on the repo.
> 😄
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/giampaolo/psutil/pull/1230#issuecomment-370254264>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAplLKt0jUBlIrtO5a7LLxeXjbSCHHWLks5tbDpWgaJpZM4SbcsS>
> .
>
-- 
Giampaolo - http://grodola.blogspot.com

jdufresne committed:
Use environment markers for conditional dependencies

Using environment markers has support among modern Python packaging
tools (setuptools, pip, wheel, etc.). Allows describing environment
specific dependencies in setup.py without using runtime logic.
